deploy/charts/jetstack-agent/README.md

```diff
@@ -2,136 +2,151 @@
 
 Jetstack Secure Agent
 
-![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.38.0](https://img.shields.io/badge/AppVersion-v1.38.0-informational?style=flat-square)
+![Version: 0.2.0](https://img.shields.io/badge/Version-0.2.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.1.39](https://img.shields.io/badge/AppVersion-v0.1.39-informational?style=flat-square)
 
 ## Additional Information
 
-The Jetstack secure agent helm chart installs the Kubernetes agent that connects to The TLS Protect For Kubernetes platform.
+The Jetstack Secure agent helm chart installs the Kubernetes agent that connects to the TLS Protect For Kubernetes (TLSPK) platform.
 It will require a valid TLS Protect for Kubernetes organisation with a license to add the new cluster.
-You should also choose a unique name for your cluster that it will appear under in the TLPK platform.
+You can sign up for a free account with up to two clusters [here](https://platform.jetstack.io/).
+You should also choose a unique name for your cluster that it will appear under in the TLSPK platform.
 
-## Installing the Chart
+## Installation:
 
-### Obtaining credentials
+Using chart installation, there are two credentials required.
 
-First obtain your service account credential, this can be done through the UI or [jsctl](https://github.com/jetstack/jsctl/releases)
+- A credential to allow helm to pull the chart from our private OCI registry.
+- An agent credential used by the agent to authenticate to TLSPK.
 
-For example with `jsctl`:
+### 1) Obtain OCI registry credentials
+
+The helm chart is an OCI chart artifact hosted on both EU and US registries:
+
+- `oci://eu.gcr.io/jetstack-secure-enterprise/charts/jetstack-agent`
+- `oci://us.gcr.io/jetstack-secure-enterprise/charts/jetstack-agent`
```

Comment:
we do have a process built in enterprise builds just pending a final review and merge here https://github.com/jetstack/enterprise-builds/pull/104

Reply:
Thanks @SpectralHiss - yes @sitaramkm please see that thread regarding this issue. The current `0.1.0` version of the chart is there because Houssem was proving out his work on publishing the chart, which may have only went to the eu registry. Hopefully once that PR is merged, all future versions will go to both registries.